webapp/src/containers/components/modals/configModal.tsx

```diff
@@ -0,0 +1,454 @@
+                                    <th>{'Channels Added to'}</th>
+                                    <th>{'Success Message'}</th>
+                                    <th>{'Options'}</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <tr>
+                                    <td>{'Button'}</td>
+                                    <td>{'Import'}</td>
+                                    <td>{'channel1, channel2, channel3'}</td>
+                                    <td>{'Welcome to your new team mate !'}</td>
+                                    <td>
+                                        <ButtonGroup
+                                            aria-label='Basic example'
+                                            className='options'
+                                        >
+                                            <OverlayTrigger
+                                                placement='top'
+                                                overlay={<Tooltip>{'Edit action'}</Tooltip>}
+                                            >
+                                                <Button onClick={handleActions}>
+                                                    <svg
+                                                        className='svg'
+                                                        xmlns='http://www.w3.org/2000/svg'
+                                                        width='20'
+                                                        height='20'
+                                                        viewBox='0 0 24 24'
+                                                        fill='none'
+                                                        stroke='#333'
+                                                        strokeWidth='1.65'
+                                                        strokeLinecap='round'
+                                                        strokeLinejoin='round'
+                                                    >
+                                                        <path d='M20 14.66V20a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2V6a2 2 0 0 1 2-2h5.34'/>
+                                                        <polygon points='18 2 22 6 12 16 8 16 8 12 18 2'/>
+                                                    </svg>
+                                                </Button>
+                                            </OverlayTrigger>
+                                            <OverlayTrigger
+                                                placement='top'
+                                                overlay={<Tooltip>{'Delete action'}</Tooltip>}
+                                            >
+                                                <Button onClick={handleDelete}>
+                                                    <svg
```

Comment:
I think we created a separate file for svgs? If not, create it now

Reply:
Created in #14